pages/dmp_knowledge_base/2_documentation_quality.md

```diff
@@ -42,12 +42,12 @@ File organisation and naming conventions should be part of the dataset descripti
 > 2a.5\
 Consider what other documentation is needed to enable re-use. This may include information on the methodology used to collect the data, analytical and procedural information, definitions of variables, units of measurement, and so on.
 
-These considerations needs to be done prior to the active phase of the research, to prepare for the information to be collected simultaneously with data collection. And the information needs to be made available as part of the archived dataset (e.g. in the 'readme'-file).
+These considerations needs to be done prior to the active phase of the research, to prepare for the information to be collected simultaneously with data collection. In addition the information needs to be made available as part of the archived dataset (e.g. embedded in the metadata or in a ReadMe-file) .
 
 > 2a.6\
 Consider how this information will be captured and where it will be recorded (for example in a database with links to each item, a ‘readme’ text file, file headers, code books, or lab notebooks).
 
-These considerations needs to be done prior to the active phase of research, to prepare for this to be taken care of simultaneously with data collection. If data documentation is recorded in a non-digital format (e.g. during laboratory or field work), explain how (parts of) the information will be digitalized.\
+If data documentation is recorded in a non-digital format (e.g. during laboratory or field work), explain how (parts of) the information will be digitalized, e.g. in ReadMe-files.\
 The information should then be documented (e.g. in the dataset's 'readme'-file) when preparing the data for archiving/publishing.
```

Comment:
```suggestion
Information on data processing and analysis steps should be included in the data documentation and kept up-to-date. This may include software versions, dependencies and workflows. In addition to parameters and settings, it is advisable to document approaches and decisions taken in the analysis process.

For more advanced use, use of workflow systems, reproducible deployment and build environments should be considered.
```